src/tools/genpolicy/rules.rego

```diff
@@ -1633,6 +1633,31 @@ ExecProcessRequest {
     print("ExecProcessRequest 3: true")
 }
 
+UpdateInterfaceRequest {
+    print("UpdateInterfaceRequest 1: input =", input)
+
+    i_interface = input.interface
```

Comment:
in rego, the standard assignment is `:=` and that's what we normally use https://www.openpolicyagent.org/docs/latest/policy-language/#equality-assignment-comparison-and-unification